packages/cli/src/commands/account/new.ts

```diff
@@ -98,7 +101,13 @@ export default class NewAccount extends BaseCommand {
     if (derivationPath) {
       derivationPath = derivationPath.endsWith('/') ? derivationPath.slice(0, -1) : derivationPath
     }
-    return derivationPath === 'eth' ? ETHEREUM_DERIVATION_PATH : derivationPath
+
+    const namedPath = DerivationPathAliases[derivationPath as keyof typeof DerivationPathAliases]
```

Comment:
could we maybe check if the key exists instead of accessing it?

also would be cool to add a test case for not using a named path

Reply:
we already have tests for when derivation path is not an alias/namedPath so i dont think a new one is needed. 

Im not sure if there is much difference between checking key exists vs checking if result is undefined